### PR TITLE
Fix/CGA-167/Don't determinate interaction wrapper as a text block

### PR DESCRIPTION
**Pull request by andrey-ponamarev**
**Related to task:**  https://oat-sa.atlassian.net/browse/CGA-167
**Description:** The COGNIA client has some items where interactions are wrap to the `div`'s element without the `grid` layout. In this case the element is determining as a text block and will not work properly in editing state.
**How to test:** 
- The taoMp extensions must be installed to support scaffolding PCI
- Download the item attached to the https://oat-sa.atlassian.net/browse/CGA-167
- In editing mode user should be able to make changes to the item by using editor

### CODE REVIEW

File: views/js/qtiCreator/widgets/item/Widget.js [hunk 1]
```diff
@@ -390,7 +391,7 @@ define([
                 subContainerBody = $subContainer.html();//get serialized body
 
             $originalTextBlock.removeAttr('data-text-block-id').html('{{_container:new}}');
-
+            console.log('PUSH SUBCONTAINERS');
```
Comment (jsconan) [inline views/js/qtiCreator/widgets/item/Widget.js:394]: Forgotten debug line?

**Review by jsconan — CHANGES_REQUESTED** [review]
It works fine.
However:
- Regarding the code, there are several calls to `console.log()`. It looks like a debug session was not cleaned up :)
- And unfortunately, the unit tests are failing.

**Review by bziondik — APPROVED** [review]
Code review only
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by jsconan — CHANGES_REQUESTED** [review]
Unfortunately, unit tests are still failing

**Comment by andrey-ponamarev** [comment]
> Unfortunately, unit tests are still failing

@jsconan we have the unit test error on the `develop` branch as well 😔
By running `grunt testall` command is not visible, but by using `grunt connect:test:keepalive --testPort=1234` it's not breaking unit tests but it's visible in console
Screenshot 2020-10-02 at 13 13 09

The problem somehow became visible after my code changes. So for now I've increased the timeout to give `ckEditor` more time to initialized

**Review by jsconan — APPROVED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Pull request merged**